backend-checks/src/main/scala/uk/gov/nationalarchives/tdr/localaws/backendchecks/checks/metadata/FakeMetadata.scala

```diff
@@ -0,0 +1,37 @@
+package uk.gov.nationalarchives.tdr.localaws.backendchecks.checks.metadata
+
+import java.nio.file.Path
+
+import uk.gov.nationalarchives.tdr.localaws.backendchecks.api.AntivirusMetadata
+
+object FakeAntivirusMetadata {
+
+  private val eicarPattern = "(eicar).*".r
+  private val virusPattern = "(test-virus).*".r
+
+  def generate(originalFileName: Path): AntivirusMetadata = {
```

Comment:
The antivirus filename logic is unchanged - I've just moved it out of the AntivirusCheck class so that all of the file name-to-metadata conversions can be seen together.